ingest/indexer/service/indexer_streaming_service.go

```diff
@@ -113,6 +116,29 @@ func (s *indexerStreamingService) publishTxn(ctx context.Context, req abci.Reque
 		txMessages := tx.GetMsgs()
 		msgType := txMessages[0].String()
 
+		// Obtain the token in before spread
+		// Related thread: https://osmosis-network.slack.com/archives/C060VCJAVBL/p1721316448977489
```

Comment:
Our Slack policy deletes the messages. Could you please summarize the context via code-comments? 🙏